### ISSUE TITLE
Error importing package with DML

### ISSUE THREAD

**Issue by ofirzaf** [microsoft/onnxruntime-genai#642]
Installed latest version 0.3.0 of the package with directml and I am getting the following import error:
```python
>>> import onnxruntime_genai
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ProgramData\miniconda3\envs\dml\lib\site-packages\onnxruntime_genai\__init__.py", line 20, in <module>
    raise e
  File "C:\ProgramData\miniconda3\envs\dml\lib\site-packages\onnxruntime_genai\__init__.py", line 10, in <module>
    from onnxruntime_genai.onnxruntime_genai import *
ImportError: DLL load failed while importing onnxruntime_genai: A dynamic link library (DLL) initialization routine failed.
```
Not sure which DLL is missing

**Comment by baijumeswani** [comment]
This is a known issue. My guess is that you're using conda on Windows?

Please see this issue for more details: https://github.com/microsoft/onnxruntime-genai/issues/555

TLDR: run `conda install conda-forge::vs2015_runtime` should fix the issue.
